OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -605,8 +605,10 @@ public void ResolveOrder(Actor self, Order order)
 				{
 					if (IsPlane)
 						self.QueueActivity(new ReturnToBase(self, Info.AbortOnResupply));
-					else
+					else if (!Info.RepairBuildings.Contains(order.TargetActor.Info.Name))
```

Comment:
Isn't HeliReturnToBase about really returning to Base and not Service pad? Then it doesn't belong there. Do you want to move it and keep the behaviour (do nothing) or you mean to look for another Repair building in HeliReturnToBase?